### PR TITLE
Fix typo 'shared'

### PR DESCRIPTION
**Pull request by R0Wi**
The current implementation of `RegistrationContext.registerService` does not recognize the `$shared`-parameter correctly because of a typo.
image

### CODE REVIEW

**Review by ChristophWurst — APPROVED** [review]
Arrrr :pirate_flag:

**Review by MorrisJobke — APPROVED** [review]
I just pushed the code style fix 👍

**Review by kesselb — APPROVED** [review]

File: tests/lib/AppFramework/Bootstrap/RegistrationContextTest.php [hunk 1]
```diff
@@ -159,4 +162,11 @@ public function testRegisterMiddleware(): void {
 			'myapp' => $app,
 		]);
 	}
+
+	public function dataProvider_TrueFalse() {
```
Comment (kesselb) [inline tests/lib/AppFramework/Bootstrap/RegistrationContextTest.php:166]: Naming is a bit uncommon.

Comment (R0Wi) [reply tests/lib/AppFramework/Bootstrap/RegistrationContextTest.php:166]: It's just the way i name it in my app tests. What would be your suggestion for data provider functions?

Comment (ChristophWurst) [reply tests/lib/AppFramework/Bootstrap/RegistrationContextTest.php:166]: `RegisterServiceData`?

Comment (kesselb) [reply tests/lib/AppFramework/Bootstrap/RegistrationContextTest.php:166]: I don't see any convention how to name data providers right now. I would prefer to not mix snake_case and camelCase but we can also merge this.

**Comment by MorrisJobke** [comment]
CI failures are unrelated and are broken on master as well 🙈

**Pull request merged**